src/StudyViewer/StudyViewer.jsx

```diff
@@ -79,6 +84,28 @@ class StudyViewer extends React.Component {
       });
     }
 
+    // sort items - order = [<field name>, <"asc" (default) or "desc">]
+    const field = order[0];
```

Comment:
i think you will need to check if `order` is specified here? since `studyViewerConfig.defaultOrderBy` can be `undefined`?